packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -240,18 +244,19 @@ export default {
     computed: {
         ...mapState(VUEX_CHECKOUT_MODULE, [
             'address',
+            'basket',
             'customer',
+            'errors',
+            'geolocation',
```

Comment:
nit/question: Should this be `geoLocation`? Not a massive deal but just noticed other times we use geolocation, we use `GEO_LOCATION` or something of the sort.

Reply:
Hey, so this isn't something I've added in, I've just changed the order of the list, so not 100% on why there's `geolocation` and `geoLocation`. Can change it if needed, but I think it would need to be another ticket.

Reply:
I believe `geolocation` is what is returned from the API, so by having it named that way in the state you can just destructure the response

Reply:
Yes, it should be in my opinion, but not as part of this PR.
I've mentioned this in this PR: https://github.com/justeat/fozzie-components/pull/748#discussion_r582921311
But I still think we should fix it.